torch/autograd/graph.py

```diff
@@ -258,8 +259,7 @@ def __exit__(self, *args: object):
 
 
 class save_on_cpu(saved_tensors_hooks):
-    """Context-manager under which tensors saved by the forward pass will be
-    stored on cpu, then retrieved for backward.
+    """Context manager under which tensors saved by the forward pass will be stored on cpu, then retrieval for backward.
```

Comment:
```suggestion
    """Context manager under which tensors saved by the forward pass will be stored on cpu, then retrieved for backward.
```

Reply:
Good catch. thanks. Fix in latest commit